core/src/main/java/org/jdbi/v3/core/statement/ParsedParameters.java

```diff
@@ -78,4 +78,22 @@ public String toString() {
             + ", parameterNames=" + parameterNames
             + '}';
     }
+
+    /**
+     * A static factory for {@link ParsedParameters} objects. It takes two
+     * arguments. The first argument is a boolean flag which indicates if
+     * statement, to which this object refers to, is using positional
+     * (<code>true</code> value) or named parameters (a <code>false</code>
+     * value). The second argument is a list which contains parameters
+     * available in SQL. In case when positional parameters are used the
+     * names list should contain only positional parameters, or more
+     * strictly, a "?" characters.
```

Comment:
typo: character (but I suspect this javadoc will be changed anyway)

Reply:
Will do.